osu.Game/Screens/Play/SubmittingPlayer.cs

```diff
@@ -234,6 +234,18 @@ protected override void StartGameplay()
             spectatorClient.BeginPlaying(token, GameplayState, Score);
         }
 
+        public override bool Pause()
+        {
+            bool wasPaused = GameplayClockContainer.IsPaused.Value;
+
+            bool paused = base.Pause();
+
+            if (!wasPaused && paused)
+                Score.ScoreInfo.PauseCount++;
```

Comment:
This is required because in real-world testing I discovered that the count could be wrong if e.g. the game was paused and then the user tabbed out, thus activating the pause-on-focus-loss logic in

https://github.com/ppy/osu/blob/321707590ab51333ce86a0b1198592705603cdb4/osu.Game/Screens/Play/Player.cs#L558-L572

In other words, `Pause()` can be called multiple times in succession and return `true` every time even if gameplay was already paused.

It's arguable that this should be changed and only the first call should return `true` but I didn't feel it wise to be changing that in this diff.